jazz_deployments-event-handler/index.js

```diff
@@ -366,7 +367,7 @@ module.exports = {
 	getTokenRequest: getTokenRequest,
 	getAuthResponse: getAuthResponse,
 	handleError: handleError,
-	checkForInterestedEvents: checkForInterestedEvents,
+	checkForInterestedEvents,
```

Comment:
i guess all the functions will have to be updated not just checkForInterestedEvents